m4/ac_path_kernel_source.m4

```diff
@@ -105,6 +105,7 @@ AC_DEFUN([AC_KERNEL_CHECKS],
   CPPFLAGS="-include $kerneldir/include/linux/kconfig.h \
             -include $kerneldir/include/linux/compiler.h \
             -D__KERNEL__ \
+            -DKBUILD_MODNAME=\"xpmem_configure\" \
```

Comment:
why "xpmem_configure" and not "xpmem" ?

Reply:
since this is only for configure step and we are hand crafting build cli, I wanted to make it clearly differentiated from standard xpmem module, when/if it pops in some logs at a later point.